packages/server/postgres/migrations/1651778831990_addMSTeamsIntegrationProvider.ts

```diff
@@ -0,0 +1,21 @@
+import {Client} from 'pg'
+import getPgConfig from '../getPgConfig'
+
+export async function up() {
+  const client = new Client(getPgConfig())
+  await client.connect()
+  await client.query(`
+  DO $$
+  BEGIN
+    ALTER TYPE "IntegrationProviderServiceEnum" ADD VALUE 'msTeams';
+  END $$;
+  `)
+  await client.end()
+}
+
+export async function down() {
+  const client = new Client(getPgConfig())
+  await client.connect()
+  await client.query(`` /* Do undo magic */)
```

Comment:
Are we able to add a down migration here? If we don't have them in our other, similar integrations, don't worry